python_ta/patches/error_messages.py

```diff
@@ -4,18 +4,193 @@
+                         'E0241': 'The class %r should not inherit from the same class multiple times.',
+                         'R0201': 'Method does not make use of \'self\', so you should rewrite the '
+                                  'method as a function and move it outside the class.',
+                         'R0205': 'Class %r inherits from \'object\', which can be omitted in Python3.',
+                         'W0211': 'Static method %r shouldn\'t contain \'self\' as the first parameter, '
+                                  'since this suggests we are expecting a class instance as the first argument.',
+                         'W0212': 'Since %s starts with an underscore, it should be considered '
+                                  '"private", and not accessed outside of the class in which it is '
+                                  'defined.',
+                         'W0221': 'Method must take the same number of arguments as %s %r method.',
+                         'W0222': 'The method parameters must have the same name, order, and default '
+                                  'arguments as %s %r method.',
+                         'W0233': 'The abstract method %r in class %r must be overridden within a '
+                                  'concrete class.',
+                         'W0231': 'Child class should call the \'__init__\' method from it\'s base class %r.',
+                         'W0233': 'Child class should call the \'__init__\' method of the parent class '
+                                  'rather than some unrelated class.',
+                         },
+        'SpecialMethodsChecker': {
+            'E0302': 'The special method %r expects to take %s parameters, but %d %s given.',
+        },
+    },
+    'pylint.checkers.exceptions': {
+        'ExceptionsChecker': {
+            'E0701': '\'except\' blocks are analyzed top to bottom, so generic exception types should '
+                     'come after specific exception types. Otherwise, the block for the specific '
+                     'exception type will never be reached. (%s)',
+            'E0702': '\'raise\' is used to raise Exceptions. Anything else will lead to an error. '
+                     '(Raising %s)',
+            'E0704': '\'raise\' may only be called without an expression inside an \'except\' block.',
+            'E0710': '\'raise\' expects an object derived from the BaseException class. Attempting to '
+                     'raise any other object will lead to an error.',
+            'E0711': '\'NotImplemented\' is strictly used as a return value for binary special methods '
+                     '(e.g __eq__), indicating that they\'re not implemented. You mean to raise '
+                     '\'NotImplementedError\', which indicates that the abstract method must be '
+                     'implemented by the derived class.',
+            'E0712': '\'except\' expects an object derived from the BaseException class. Attempting to '
```

Comment:
object -> class

derived from -> subclass of

Remove the second sentence (it is unrelated)